template/subscription_template.json

```diff
@@ -34,12 +34,13 @@
     }
   },
   "variables": {
-    "sbVersion": "2017-04-01"
+    "sbVersion": "2017-04-01",
+    "subscriptionName": "[concat(parameters('serviceBusNamespaceName'), '/', parameters('serviceBusTopicName'), '/', parameters('serviceBusSubscriptionName'))]"
```

Comment:
It would be best to name it `fullSubscriptionName` so that it would be clear how it differs from `serviceBusSubscriptionName`